mac/watchman_visualize.mac

```diff
@@ -1,10 +1,25 @@
 /glg4debug/glg4param omit_muon_processes 1.0
 /glg4debug/glg4param omit_hadronic_processes 1.0
 
+#test comment
+#second test comment
 /rat/db/set DETECTOR experiment "Watchman"
 /rat/db/set GEO[inner_pmts] mu_metal 0
+/rat/db/set GEO[inner_pmts] light_cone 1
+#/rat/db/set GEO[inner_pmts] mu_metal_surface "mirror"
```

Comment:
Is mu_metal_surface used also for the light concentrator?

Reply:
i have removed mu_metal options from visualization macros since they are not applicable. all light cone options will be removed from watchman_visualize.mac and watchman_visualize_geofile.mac and put in a separate macro called watchman_visualize_geofile_lightcones.mac, such that light cones can be visualized and explored using that macro specifically and not be spread randomly throughout other macros.